recipes/openssl/3.x.x/conanfile.py

```diff
@@ -338,6 +338,11 @@ def _get_default_openssl_dir(self):
             return "/etc/ssl"
         return os.path.join(self.package_folder, "res")
 
+    def _adjust_path(self, path):
+        if self._use_nmake:
+            return path.replace("\\", "/")
+        return unix_path(self, path)
```

Comment:
I am able to build in MinGW, without a subsystem, and without NMake, but it still requires the ``.replace("\\", "/")``, which doesn't seem covered by this ``adjust_path``, so it seems my fix in https://github.com/conan-io/conan-center-index/pull/20448 is still good and necessary, not covered by this PR

Reply:
I don't understand. You can't build with MinGW without a subsystem by design of this recipe.